monitoring_service/test/conftest.py

```diff
@@ -1,5 +1,11 @@
-from gevent import monkey
-monkey.patch_all()
+"""
+For some reason isort changes the behaviour of this file. Might have something
+to do with monkey.patch_all.
+
+isort:skip_file
```

Comment:
Yes, you always have to monkeypatch before importing other stuff: see https://github.com/raiden-network/raiden-pathfinding-service/blob/master/pathfinder/cli.py#L2